Dockerfile

```
@@ -0,0 +1,14 @@
+FROM rust:alpine as builder
+RUN apk add clang musl-dev
+
+COPY . /tmp
+WORKDIR /tmp
+
+RUN cargo --version
+RUN cargo build --release --target x86_64-unknown-linux-musl
+
+FROM scratch as base
```

I would rebuild put it back to alpine

```suggestion
FROM alpine as base
```